beakerx/beakerx/static/html/settings_tab.html

```diff
@@ -38,7 +46,11 @@
     <div class="beakerx_container">
         <form id="beakerx_jvm_settings_form" class="form-horizontal">
             <fieldset id="default_options">
-                <legend>JVM Options:</legend>
+                <legend>JVM Options: <span id="sync_indicator">
```

Comment:
please move the sync indicator to the bottom, where the "Result" string is, and where the errors are shown.